custom_components/powercalc/data/model_schema.json

```diff
@@ -83,7 +87,13 @@
         },
         "max_watt": {
           "type": "number"
-        }
+        },
+	"calibrate": {
```

Comment:
Did you test if this was working correctly already? Maybe some work need to be done on correctly mapping this config to the python dictionary.